SeaPublicWebsite/Views/Cookie/CookieSettings.cshtml

```diff
@@ -0,0 +1,91 @@
+﻿@model SeaPublicWebsite.Models.Cookies.CookieSettingsViewModel
+@{
+    ViewBag.Title = $"Cookies on our service";
+}
+
+<div class="govuk-grid-row cookie-settings-page">
+    <div class="govuk-grid-column-two-thirds">
+        @if (Model.ChangesHaveBeenSaved)
+        {
+            <div class="govuk-notification-banner govuk-notification-banner--success" role="alert" aria-labelledby="govuk-notification-banner-title" data-module="govuk-notification-banner">
+              <div class="govuk-notification-banner__header">
+                <h2 class="govuk-notification-banner__title" id="govuk-notification-banner-title">
+                  Success
+                </h2>
+              </div>
+              <div class="govuk-notification-banner__content">
+                <p class="govuk-notification-banner__heading">
+                  You’ve set your cookie preferences.
+                </p>
+              </div>
+            </div>
+        }
+
+        <form method="POST" action="@Url.Action("CookieSettings_Post", "Cookie")">
+            @(Html.AntiForgeryToken())
```

Comment:
This shouldn't be necessary in .net core 6. Please remove this line and check that the token is still generated on the page.